apps/auth/service.py

```diff
@@ -146,13 +146,9 @@ def __can_clear_sessions(self, user):
 
         Operation is invalid if one of the below is True:
             1. Check if the user exists.
-            2. Check if the user is clearing his/her own sessions.
 
         :return: error message if invalid.
         """
 
         if not user:
             return "Invalid user to clear sessions."
-
-        if str(user[config.ID_FIELD]) == str(flask.g.user[config.ID_FIELD]):
```

Comment:
just not sure if we should drop the current session, how is that handled?

Reply:
Actually the problem was that, logged user can not clear their own season, so when I investigated I found we have a prebuilt functionality logged user can not clear their own session. so here I remove that condition.

Reply:
that part is fine, I'm just not sure if it should drop all user session but the current one so the user won't end up logged out of the system

Reply:
So you mean we have to clear all session of active user except current session, so user would not logged out of the system.

Reply:
exactly

Reply:
Actually, I looked in to the code and I couldn't distinguish between the current and other sessions of user? can you please provide me some suggestions on it. :)